packages/esm-ugandaemr-app/src/pages/system-info/system-info.types.tsx

```diff
@@ -0,0 +1,63 @@
+export interface systemInfo {
+  'SystemInfo.title.moduleInformation': SystemInfoTitleModuleInformation;
+}
+
+export interface SystemInfoTitleModuleInformation {
+  'SystemInfo.Module.repositoryPath': string;
+  'Order Templates': string;
+  'Database Backup Module': string;
+  'ID Generation': string;
+  'Allergy UI Module': string;
+  'EMR API Module': string;
+  'Patient Queueing': string;
+  'HTML Form Entry UI Framework Integration Module': string;
+  'App Framework Module': string;
+  'Reporting REST': string;
+  'Reference Metadata Module': string;
+  UgandaemrSync: string;
+  'App UI Module': string;
+  'Metadata Deploy': string;
+  'Reporting Compatibility': string;
+  'Bed Management Module': string;
+  'O3 Forms': string;
+  'HTML Widgets': string;
+  'Address Hierarchy': string;
+  'Data Entry Statistics': string;
+  'Registration Core Module': string;
```

Comment:
are these how it's being defined in the api? typically the format is 

interface SystemInfoTitleModuleInformation {
  systemInfo: string;
  idGeneration: string;
...
}

Reply:
The endpoint returns the information in that pattern so it is how i defined it.